shared/data/fingerprint-protection.js

```diff
@@ -23,7 +23,7 @@
     }
 
     // Property values to be set and their original values.
-    const fingerprintPropertyValues = {
+    let fingerprintPropertyValues = {
```

Comment:
nit - if we are not overwriting whole `fingerprintPropertyValues` with a new object (and just adding a property to existing object), this should still be const.